### DISCUSSION TITLE
Feature Request - File Browser (Accessory type/payload?)

### DISCUSSION THREAD

**Discussion by PaulC2018** [IBM/mac-ibm-notifications#145] [Ideas]
Hi, 

Would do people think about having a function for endusers to browse external files/folders? 

Cocadialog use to have the functionality `fileselect` that did this. 

Use scenarios - This could be handy if you wanted to give the enduser an options open to an application without having to create a pre defined list for a dropdown field. 

Allowing them to select a file eg, screen capture, log file etc. that could be provided to a support party in an email as either a file path for reference or an attachment to additional email client.

**Comment by SMartorelli** [comment]
Thanks for the Feature Request. I created the issue starting from this discussion. [#145](https://github.com/IBM/mac-ibm-notifications/issues/146)
